# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rarpd/tftpboot-dir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rarpd/rarpd.c.orig	2006-01-14 03:48:12.000000000 +0100
+++ rarpd/rarpd.c	2006-01-14 05:20:18.000000000 +0100
@@ -42,7 +42,7 @@ int only_ethers;
 int all_ifaces;
 int listen_arp;
 char *ifname;
-char *tftp_dir = "/tftpboot";
+char *tftp_dir = "/srv/tftp";
 
 void usage(void) __attribute__((noreturn));
 
